plutus-playground-client/src/MainFrame.purs

```diff
@@ -74,19 +76,19 @@ import View as View
 import Wallet.Emulator.Wallet (Wallet(Wallet))
 import Web.HTML.Event.DataTransfer as DataTransfer
 
-mkSimulatorWallet :: Array KnownCurrency -> Int -> SimulatorWallet
+mkSimulatorWallet :: Array KnownCurrency -> BigInt -> SimulatorWallet
 mkSimulatorWallet currencies walletId =
```

Comment:
For wallet ids I think Int would suffice. @j-mueller, right?

Reply:
Yes `Int` would be enough but we are using `Integer` for everything on the Haskell side (incl. wallet identifiers), so it makes sense to do the same here, just to be consistent.